kotlin/example-app/build.gradle.kts

```diff
@@ -0,0 +1,72 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.jetbrains.kotlin.android)
+    alias(libs.plugins.compose.compiler)
+}
+
+android {
+    namespace = "io.mosip.injivcrenderer"
+    compileSdk = 34
+
+    defaultConfig {
+        applicationId = "io.mosip.injivcrenderer"
+        minSdk = 24
+        targetSdk = 34
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
```

Comment:
we should try with java21

Reply:
We can have example app as Java 21, but since library using Java 17, for consistency, we will use Java17. I will update it from Java 8 to Java 17